pkg/products/observability/prometheusRules.go

```diff
@@ -262,9 +262,9 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 				{
 					Alert: "RHOAMCSVRequirementsNotMet",
 					Annotations: map[string]string{
-						"message": "RequirementsNotMet for CSV '{{$labels.name}}' in namespace '{{$labels.namespace}}'. Phase is {{$labels.phase}}",
+						"message": "RequirementsNotMet for CSV '{{$labels.name}}' in namespace '{{$labels.exported_namespace}}'. Phase is not succeeded",
 					},
-					Expr:   intstr.FromString(fmt.Sprintf("csv_abnormal{phase=~'Pending|Failed',exported_namespace=~'%s.*'}", r.Config.GetNamespacePrefix())),
+					Expr:   intstr.FromString(fmt.Sprintf(`csv_succeeded{exported_namespace=~"%s.*"} != 1`, r.Config.GetNamespacePrefix())),
```

Comment:
@cecobask feel like this will fire anytime we are doing an install or upgrade, could we add a time component to it i.e. the  csv has been in an unsuccessful state for more than half an hour

Reply:
@austincunningham I didn't witness the alert firing during the upgrade scenario. I can check what happens in a fresh install scenario to confirm. I've added a time frame of `15m` to it which seems like a reasonable timeframe, but I can update it to `30m` if requested

Reply:
The alert did not trigger or fire during a fresh install scenario either 👍🏻